server/src/services/plugins.service.ts

```diff
@@ -238,8 +238,8 @@ export class PluginsService {
 
     try {
       // validate manifest and operations as JSON files
-      const validManifest = JSON.parse(manifest.toString()) ? manifest : null;
-      const validOperations = JSON.parse(operations.toString()) ? operations : null;
+      // const validManifest = JSON.parse(manifest.toString()) ? manifest : null;
```

Comment:
shall we remove the entire try catch if its not used?

Reply:
I am not very sure about this part. I commented this code only because the lint was failing

Reply:
I will confirm with @arpitnath

Reply:
I will remove the comments and will rebase with the `develop` once @arpitnath's lint PR merged